ui-v2/src/components/deployments/deployment-flow-link.tsx

```diff
@@ -11,7 +11,7 @@ export const DeploymentFlowLink = ({ flowId }: DeploymentFlowLinkProps) => {
 	const { data: flow } = useSuspenseQuery(buildFLowDetailsQuery(flowId));
 
 	return (
-		<div className="flex items-center gap-1 text-sm">
+		<div className="flex items-center gap-1 text-xs">
```

Comment:
Also for future us. we'll need a `<FlowLink />` component for other pages. So we might want to make that component and have this `<DeploymentFlowLink />` just be the suspense/query wrapper around it.